content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -109,35 +109,50 @@ To set up a Uruware account, first be sure to have a valid Odoo subscription. Th
 proceeding steps.
 
 Locate the Uruware credentials settings by navigating to the :menuselection:`Accounting -->
-Configuration --> Settings`. Scroll down to the :guilabel:`Uruguayan Localization` section and click
-on :guilabel:`Create Uruware Account`.
+Configuration --> Settings`. Scroll down to the :guilabel:`Uruguayan Localization` section, select
+the environment (Production or Testing) and click on :guilabel:`Create Uruware Account`.
```

Comment:
We can use :guilabel: here for these radio buttons :)

@fvz-odoo What about the "Demo" option? How is this option used?

```suggestion
the environment (:guilabel:`Production` or :guilabel:`Testing`) and click on :guilabel:`Create
Uruware Account`.
```

Reply:
> We can use :guilabel: here for these radio buttons :)
> 
> @fvz-odoo What about the "Demo" option? How is this option used?

Demo mode does not work with Uruware, it is only for internal validations. We mention a bit after what it's for in line 155. Do you think it is worth to mention that demo mode does not need an Uruware account?

Reply:
I see! Yes perhaps it would be a good idea to add onto the line 155 description: "Using :guilabel:`Demo mode` does not require a Uruware account."

Reply:
Hey @samueljlieber 

Perfect, I did it here d9c17bc500b6cd04c56c1353e84a9c8d1a97479b

Let me know what you think. Cheers!